Added description of per-workspace PVC strategy (#636)
codenvy/che-docs: src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc

[width="100%",cols="25%,25%,25%,25%",options="header",]
|===
|strategy |details |pros |cons

> |*unique (default)* | One PVC per workspace's volume |Storage isolation |An undefined number of PVs is required

Reviewer: ```suggestion
|*unique (default)* | One PVC per workspace volume |Storage isolation |An undefined number of PVs is required
```

|*common* | One PVC for all workspaces in one {k8s-namespace}

> Sub-paths pre-created |Easy to manage and control storage |Workspaces must be in a separate {k8s-namespace} if PV doesn't support ReadWriteMany Access Mode

Reviewer: ```suggestion
Sub-paths pre-created |Easy to manage and control storage |Workspaces must be in a separate {k8s-namespace} if PV does not support ReadWriteMany (RWX) access mode
```

|*per-workspace* | One PVC for one workspace

> Sub-paths pre-created |Easy to manage and control storage |Workspace containers must all be in one Pod if PV doesn't support ReadWriteMany Access Mode

Reviewer: ```suggestion
Sub-paths pre-created |Easy to manage and control storage |Workspace containers must all be in one pod if PV does not support ReadWriteMany (RWX) access mode
```

Another restriction is that only pods in the same namespace can use the same PVC.  The `CHE_INFRA_KUBERNETES_PROJECT` environment variable should not be empty.  It should be either the Che server namespace where objects can be created with the Che service account (SA) or a dedicated namespace where a token or a username and password need to be used.

[id="per-workspace-pvc-strategy"]

> === PerWorkspace PVC Strategy

Reviewer: ```suggestion
=== Per porkspace PVC strategy
```

Author: Will change to `Per workspace PVC strategy`

To define the unique strategy, set `CHE_INFRA_KUBERNETES_PVC_STRATEGY` to `per-workspace`.

[id="how-the-per-workspace-pvc-strategy-works"]

> ==== How the PerWorkspace PVC strategy works

Reviewer: ```suggestion
==== How the per workspace PVC strategy works
```

> Basically it works in the same ways as Common PVC Strategy.

Reviewer: ```suggestion
It works similarly to the common PVC strategy.
```

> The only different that all workspace volumes use the same PVC to store data (projects and workspace logs by default and whatever additional link:volumes.html[volumes] that a user can define.) but not all workspaces

Reviewer: ```suggestion
The only difference is that all workspace volumes use the same PVC to store data (projects and workspace logs by default and any additional link:volumes.html[volumes] that a user can define) but not all workspaces
```
Could you please explain what the end of the sentence means ? (`but not all workspaces`) What is it in relation to?

Author: I wanted to make clear difference between `Per workspace` and `Common` strategies
So, `The only difference is that **all workspace volumes** use the same PVC ... but **not all workspaces**.`
Agree that it looks confusing. Should I just remove `but not all workspaces` or would you help me rephrase it?

Reviewer: Got it, thanks. How about:

_"The only difference is that all workspace volumes (but not all workspaces) use the same PVC to store data (projects and workspace logs by default and any additional link:volumes.html[volumes] that a user can define)."_

Author: LGTM, thanks
